.github/workflows/macchina.yml

```diff
@@ -87,6 +87,13 @@ jobs:
           args: --target=${{ matrix.target }}
           use-cross: ${{ matrix.cross }}
 
+      - name: Test
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --target=${{ matrix.target }}
+        if: ${{ !matrix.cross }}
+
```

Comment:
There's no need for this step (at least not right now) as macchina has no unit tests :(